### PR TITLE
Bug fix: ./hack/install-etcd.sh is not work on Arm64 platform

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
We need to fix it so that we can install etcd on Arm64 platform.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
We can use it to install etcd for e2e test on Arm64 platform.

**Does this PR introduce a user-facing change?**:
NONE

```release-note
NONE
```

### CODE REVIEW

File: hack/lib/etcd.sh [hunk 1]
```diff
@@ -118,14 +141,17 @@ kube::etcd::install() {
...
       kube::util::download_file "${url}" "${download_file}"
       tar xzf "${download_file}"
-      ln -fns "etcd-v${ETCD_VERSION}-linux-amd64" etcd
+      ln -fns "etcd-v${ETCD_VERSION}-linux-${arch_linux}" etcd
       rm "${download_file}"
     fi
     kube::log::info "etcd v${ETCD_VERSION} installed. To use:"
     kube::log::info "export PATH=$(pwd)/etcd:\${PATH}"
+    if [[ ${arch_linux} != "amd64" ]]; then
+      kube::log::info "export ETCD_UNSUPPORTED_ARCH=${arch_linux}"
```
Comment (dims) [inline hack/lib/etcd.sh]: Looks like this is coming from https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/supported-platform.md#32-bit-and-other-unsupported-systems Right?

Comment (dims) [reply hack/lib/etcd.sh]: Can you please add a note for future?

**Review by dixudx — APPROVED** [review]
lgtm overall.

File: hack/lib/etcd.sh (hunk 1 above)
Comment (lubinsz) [reply hack/lib/etcd.sh]: Done

**Comment by lubinsz** [comment]
@dixudx @dims @BenTheElder
Hi all,
Any comments?

**Review by dixudx — APPROVED** [review]
/lgtm

File: hack/lib/etcd.sh [hunk 2]
```diff
@@ -105,24 +105,33 @@ kube::etcd::cleanup() {
...
+    arch=$(kube::util::host_arch)
+
     cd "${KUBE_ROOT}/third_party"
-    os=$(uname | tr "[:upper:]" "[:lower:]")
     if [[ $(readlink etcd) == etcd-v${ETCD_VERSION}-${os}-* ]]; then
       return  # already installed
     fi
+
     if [[ ${os} == "darwin" ]]; then
+      if [[ ${arch} != "amd64" ]]; then
```
[the remark this answers is not in the document]
Comment (dixudx) [reply hack/lib/etcd.sh]: @BenTheElder Good point. I think this check is redundant.

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by lubinsz** [comment]
@dims 
Can you approve it?
Thanks.

**Pull request merged**